### PR TITLE
Task name, description, and 'helpUrl' consistency

### PR DESCRIPTION
**Pull request by DavidStaheli**

### CODE REVIEW

File: Tasks/NpmV1/task.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "Npm",
     "friendlyName": "npm",
     "description": "Install and publish npm packages, or run an npm command. Supports npmjs.com and authenticated registries like Package Management.",
-    "helpUrl": "https://go.microsoft.com/fwlink/?LinkID=613746",
```
Comment (zjrunner) [inline Tasks/NpmV1/task.json]: Is the preferred style no-fwlinks?  It sure makes PR review easier, but it means any doc changes require a release...which means trouble for onprem potentially.

Comment (DavidStaheli) [reply Tasks/NpmV1/task.json]: Good question.  There are so many tasks that creating an fwlink for each that is opaque seemed difficult to manage.  The doc URLs are treated as stable - they are never changed without also adding a redirect to the changed content.

File: Tasks/AppCenterDistributeV3/Strings/resources.resjson/en-US/resources.resjson [hunk 2]
```diff
@@ -29,9 +29,9 @@
   "loc.input.help.releaseNotesFile": "Select a UTF-8 encoded text file which contains the Release Notes for this version.",
   "loc.input.label.isMandatory": "Require users to update to this release",
   "loc.input.label.destinationType": "Release destination",
-  "loc.input.label.destinationGroupIds": "Distribution Group IDs",
+  "loc.input.label.destinationGroupIds": "Destination IDs",
```
Comment (DennisPan) [inline Tasks/AppCenterDistributeV3/Strings/resources.resjson/en-US/resources.resjson]: Are these changes intentional?

Comment (DavidStaheli) [reply Tasks/AppCenterDistributeV3/Strings/resources.resjson/en-US/resources.resjson]: @DennisPan this file is auto-generated.  Since the file is changed in this PR, that means someone changed this string in `task.json` but didn't also change the string in this file (`resources.resjson`) in a previous PR.

File: Tasks/DotNetCoreInstallerV1/task.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
     "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
     "name": "DotNetCoreInstaller",
-    "friendlyName": ".NET Core SDK/Runtime Installer",
-    "description": "Acquires a version of the .NET Core SDK/Runtime from the internet or the local cache and adds it to the PATH. Use this task to change the version of .NET Core used in subsequent tasks.",
-    "helpUrl": "https://aka.ms/AA4ihfi",
+    "friendlyName": ".NET Core sdk/runtime installer",
+    "description": "Acquire a specific version of the .NET Core SDK from the internet or local cache and add it to the PATH",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer",
```
Comment (sachinma) [inline Tasks/DotNetCoreInstallerV1/task.json]: This still points to v0 documentation. Should we update the same page or add another one with the update?

Comment (DavidStaheli) [reply Tasks/DotNetCoreInstallerV1/task.json]: @sachinma I'm glad you noticed that.  Until a better pattern is established, the content team's current approach is to update the same doc page to the latest info about the task.

Comment (sachinma) [reply Tasks/DotNetCoreInstallerV1/task.json]: Ok David. I will work with @pulkitaggarwl  to get the .Net core documentation updated to the latest version.

File: Tasks/PyPIPublisherV0/task.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
     "id": "2d8a1d60-8ccd-11e7-a792-11ac56e9f553",
     "name": "PyPIPublisher",
-    "friendlyName": "PyPI Publisher",
-    "description": "Create and upload an sdist or wheel to a PyPI-compatible index using Twine.",
+    "friendlyName": "PyPI publisher",
+    "description": "Create and upload an sdist or wheel to a PyPI-compatible index using twine",
```
Comment (lkillgore) [inline Tasks/PyPIPublisherV0/task.json]: Shouldn't Twine be capitalized?

File: Tasks/NuGetPublisherV0/task.json [hunk 5]
```diff
@@ -131,17 +131,17 @@
         }
     },
     "messages": {
-        "Warning_NoConfigForOldNuGet": "To connect to NuGet feeds hosted in your Azure Pipelines/TFS project collection with NuGet 3.1 or below, edit your build pipeline to specify a path to a NuGet.config containing the package sources you wish to use.",
-        "Warning_NoConfigForNoCredentialProvider": "To connect to NuGet feeds hosted in your Azure Pipelines/TFS project collection on this build agent, edit your build pipeline to specify a path to a NuGet.config containing the package sources you wish to use.",
+        "Warning_NoConfigForOldNuGet": "To connect to NuGet feeds hosted in your Azure Pipelines organization with NuGet 3.1 or below, edit your build pipeline to specify a path to a NuGet.config containing the package sources you wish to use.",
```
Comment (lkillgore) [inline Tasks/NuGetPublisherV0/task.json]: 'Azure DevOps organization'?

File: Tasks/PyPIPublisherV0/task.json (hunk 4 above)
Comment (DavidStaheli) [reply Tasks/PyPIPublisherV0/task.json]: I changed it because its [product homepage](https://pypi.org/project/twine/) consistently refers to the tool in its lowercase form.  Please shout if you disagree.

Comment (lkillgore) [reply Tasks/PyPIPublisherV0/task.json]: image
Seems inconsistent w/in the page.  I don't have strong feelings on the matter.  If I were to pick one over the other, I personally would capitalize it.
